.github/workflows/build.yml

```diff
@@ -21,7 +21,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest]
-        php: ['7.4', '8.0', '8.1']
+        php: ['8.0', '8.1']
```

Comment:
🥲 
First line changed that I see and it already involves a lot of work.
@Naktibalda If we are going to drop support for PHP 7.4 in the release of Codeception 5 I have to submit another couple of PRs updating the codebase to PHP 8.0.
Do you want me to do it before this is merged (which might force you to rebase) or do you want me to keep checking and do this including the changes from this PR?

Reply:
I really don't want to rebase this branch after enormous change in 5.0, too many files were moved.
Could you wait a few days?

Reply:
Okay no problem.

Reply:
@TavoNiievez You are welcome to update 5.0 branch now.